auction-app-backend/src/main/java/com/example/auctionapp/entity/ProductImageEntity.java

```diff
@@ -0,0 +1,73 @@
+package com.example.auctionapp.entity;
+
+import com.example.auctionapp.model.ProductImage;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import org.hibernate.annotations.GenericGenerator;
+
+import java.util.UUID;
+
+@Entity
+@Table(name = "product_image", schema="auction_app")
+public class ProductImageEntity {
+    @Id
+    @GeneratedValue(generator = "UUID")
+    @GenericGenerator(name = "UUID", strategy = "org.hibernate.id.UUIDGenerator")
+    @Column(name = "image_id")
+    private UUID imageId;
+
+    @Column(name = "image_url")
+    private String imageUrl;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "product_id")
+    private ProductEntity productEntity;
```

Comment:
Check can you do it by using OneToMany relationship in product entity, because you want to have images there.